snuba/admin/notifications/slack/client.py

```diff
@@ -25,12 +28,11 @@ def post_message(
             resp = requests.post(
                 "https://slack.com/api/chat.postMessage", headers=headers, json=message,
             )
-        except Exception:
-            # todo: log non 200 failures
-            return
+        except Exception as exc:
+            logger.error(exc, exc_info=True)
 
         # todo: slack is annoying be a 200 could still be a failed case
         # you have to check the "ok" param in the response, so we should
         # check for ok: False and log those failures too.
-        print(resp.status_code)
-        print(resp.content)
+        if resp.status_code != 200:
```

Comment:
So slack is annoying in that they return 200s even where there is an error. So you have to check th `ok` param to know whether there was an error or not. I think we wanna have some logic that is similar to: https://github.com/getsentry/sentry/blob/601f829c9246ae73c8169510140fd7f47fc6dfc3/src/sentry/integrations/slack/client.py#L36-L53
so that we can log the actual error

Reply:
☝️  That's great context to put in a comment!

Reply:
Meh. Ok copied that code snippet over.